packages/plugins/src/nuxt/index.ts

```diff
@@ -1,9 +1,10 @@
 import { addComponent, defineNuxtModule } from '@nuxt/kit'
 
 import type { } from '@nuxt/schema' // workaround for TS bug with "phantom" deps
-import * as components from 'radix-vue'
+import { components as allComponents } from '../../../radix-vue/constant/components'

```

Comment:
This path would not work for packages install via npm. Cause we are not bundling this constant.